Step03/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
```

Comment:
can you maintain only one .gitignore file ? you can merge contents from other .gitignore files into this file and maintain only one file.